src/www/ui/api/index.php

```diff
@@ -269,6 +269,7 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->get('', LicenseController::class . ':getAllLicenses');
     $app->post('/import-csv', LicenseController::class . ':handleImportLicense');
     $app->post('', LicenseController::class . ':createLicense');
+    $app->put('/verify/{shortname:[\\w\\- \\.]+}', LicenseController::class . ':verifyLicense');
```

Comment:
Would recommend using following regex for shortname as it can literally be anything as of now.
```suggestion
    $app->put('/verify/{shortname:.+}', LicenseController::class . ':verifyLicense');
```

Reply:
Done